groovy-language-server/src/main/java/com/palantir/ls/server/groovy/GroovyLanguageServer.java

```diff
@@ -14,19 +14,19 @@
  * limitations under the License.
  */
 
-package com.palantir.ls.server.groovy;
```

Comment:
moved this package since it seemed silly to have `language server....server`, but happy to move it back